cns/service/main.go

```diff
@@ -658,13 +659,18 @@ func main() {
 		}
 		defer endpointStoreLock.Unlock() // nolint
 
-		err = platform.CreateDirectory(endpointStoreLocation)
+		endpointStorePath := endpointStoreLocation
+		if runtime.GOOS == "windows" {
+			endpointStorePath = os.Getenv("CNSStoreFilePath")
+		}
+		err = platform.CreateDirectory(endpointStorePath)
```

Comment:
why do we need this? who sets `CNSStoreFilePath`

Reply:
It's from the CNS configMap written by AKS-RP. We need it since the path for Windows is different than Linux

Reply:
if cns configmap, why its getting from env? shouldn't this be part of cnsconfig field?

Reply:
This path is not within CNS CM. We have to read it from DS. It seems this is the way to read from CNS DS confing.

Reply:
This variable basically getting passed as an arg on CNS DS.